network-errors/src/commonMain/kotlin/dev/icerock/moko/network/errors/NetworkExceptionMappers.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.moko.network.errors
+
+import dev.icerock.moko.errors.mappers.BasicException
+import dev.icerock.moko.errors.mappers.ExceptionMappersStorage
+import dev.icerock.moko.network.exceptions.ErrorException
+import dev.icerock.moko.network.exceptions.ValidationException
+import dev.icerock.moko.network.isNetworkConnectionError
+import dev.icerock.moko.resources.desc.CompositionStringDesc
+import dev.icerock.moko.resources.desc.StringDesc
+import dev.icerock.moko.resources.desc.desc
+
+/**
+ * Registers all default exception mappers of the network module to [ExceptionMappersStorage].
+ */
+fun ExceptionMappersStorage.registerAllNetworkMappers(): ExceptionMappersStorage {
+    return condition(
+        condition = ::networkConnectionErrorCondition,
+        mapper = ::networkConnectionErrorStringDescMapper
+    ).register(::networkErrorExceptionStringDescMapper)
+        .register(::validationExceptionStringDescMapper)
+}
+
+/**
+ * Checks whether the input [exception] is a network connection error.
+ */
+fun networkConnectionErrorCondition(exception: BasicException): Boolean {
```

Comment:
what is `BasicException`?

Reply:
hmm for what it added? every person who will see this code will ask "what is BasicException?"...why not use just `Throwable` that anyone known?